app/src/main/java/ksc/campus/tech/kakao/map/views/KakaoMapFragment.kt

```diff
@@ -54,22 +100,80 @@ class KakaoMapFragment : Fragment() {
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        initiateViews(view)
         initiateKakaoMap(view)
+        initiateViewModelCallbacks()
         super.onViewCreated(view, savedInstanceState)
     }
 
     override fun onResume() {
-        kakaoMap.resume()
+        kakaoMapView.resume()
         super.onResume()
     }
 
     override fun onPause() {
-        kakaoMap.pause()
+        kakaoMapView.pause()
         super.onPause()
     }
 
     override fun onDestroyView() {
-        kakaoMap.finish()
+        kakaoMapView.finish()
         super.onDestroyView()
     }
+
+    private fun restoreMarker(){
+        updateSelectedLocation(viewModel.selectedLocation.value)
+    }
+
+    private fun restorePosition(kakaoMap: KakaoMap){
+        moveCamera(kakaoMap, viewModel.cameraPosition.value?:MapViewRepository.initialCameraPosition)
+    }
+
+    private fun updateSelectedLocation(locationInfo: LocationInfo?){
+        if (locationInfo == null) {
+            clearLabels()
+        } else {
+            changedSelectedPosition(LatLng.from(locationInfo.latitude, locationInfo.longitude))
+        }
+    }
+
+    private fun changedSelectedPosition(coordinate: LatLng){
```

Comment:
```suggestion
    private fun changeSelectedPosition(coordinate: LatLng){
```
과거형보다는 평서문이 컨벤션에 더 맞겠네요